### PR TITLE
https://issues.redhat.com/browse/ACM-8583

### PR DESCRIPTION
**Pull request by jc-berger**

### CODE REVIEW

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 1]
```diff
@@ -60,6 +60,10 @@ To specify the details about running the Ansible template while creating a templ
 . If you want to initiate any Ansible jobs before the cluster is installed, select *Add an Automation template* in the _Pre-install Automation templates_ section.
 
 . Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags`, and workflow types.
++
+* Use the `Extra variables` field to pass data to the `AnsibleJob` resource in the form of `key=value` pairs. 
+
+* Special keys `cluster_deployment` and `machine_pool` are passed automatically as extra variables, containing data on general information about the cluster and details about the worker nodes being created in the cluster.
```
Comment (fxiang1) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Sorry @jc-berger I made the mistake of mentioning `machine_pool` it should be `install_config` instead.
Please change this line to:

* Special keys `cluster_deployment` and `install_config` are passed automatically as extra variables, containing data on general information about the cluster and details about the cluster install configuration.

**Review by fxiang1 — CHANGES_REQUESTED** [review]

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc (hunk 1 above)
Comment (jc-berger) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Thanks @fxiang1 ! I've applied your feedback. Would you please let me know if you think it looks good to merge now?

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks Jake for the quick change!

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 2]
```diff
@@ -60,6 +60,10 @@ To specify the details about running the Ansible template while creating a templ
 . If you want to initiate any Ansible jobs before the cluster is installed, select *Add an Automation template* in the _Pre-install Automation templates_ section.
 
 . Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags`, and workflow types.
++
+* Use the `Extra variables` field to pass data to the `AnsibleJob` resource in the form of `key=value` pairs.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]:
```suggestion
* Use the *Extra variables* field to pass data to the `AnsibleJob` resource in the form of `key=value` pairs. 
```

Comment (cadawson) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: IBM style has field names as bold, but we might not always use it that way. If you see this somewhere else in the file, please make it consistent.

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 3]
```diff
@@ -60,6 +60,10 @@ To specify the details about running the Ansible template while creating a templ
 . If you want to initiate any Ansible jobs before the cluster is installed, select *Add an Automation template* in the _Pre-install Automation templates_ section.
 
 . Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags`, and workflow types.
++
+* Use the `Extra variables` field to pass data to the `AnsibleJob` resource in the form of `key=value` pairs. 
+
+* Special keys `cluster_deployment` and `install_config` are passed automatically as extra variables, containing data on general information about the cluster and details about the cluster install configuration.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]:
```suggestion
* Special keys `cluster_deployment` and `install_config` are passed automatically as extra variables. They contain general information about the cluster and details about the cluster installation configuration. 
```

Comment (cadawson) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Just a thought here to break this up.

**Review by cadawson — APPROVED** [review]
Looks good. A couple of minor suggestions to consider. Thanks!

**Pull request merged**